src/listener_message.rs

```diff
@@ -15,18 +15,11 @@
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
 
-use sodiumoxide::crypto::sign::PublicKey;
 use socket_addr::SocketAddr;
 
+pub const REQUEST_MAGIC_CONSTANT: [u8; 4] = ['E' as u8, 'C' as u8, 'H' as u8, 'O' as u8];
```

Comment:
> Hope that helps.

Yes, it helped. Thanks.

So... do I just need to convert this byte sequence into an empty struct that implements `rustc_serialize::Encodable`?